pkg/admissioncontroller/webhooks/auth/seed/graph/graph_test.go

```diff
@@ -868,6 +921,16 @@ var _ = Describe("graph", func() {
 			paths[VertexTypeBackupEntry] = append(paths[VertexTypeBackupEntry], pathExpectation{VertexTypeBackupEntry, backupEntry1.Namespace, backupEntry1.Name, VertexTypeSeed, "", *backupEntry1.Spec.SeedName, BeTrue()})
 		}()
 		wg.Add(1)
+		go func() {
+			defer wg.Done()
+			bastion1Copy := bastion1.DeepCopy()
+			bastion1.Spec.SSHPublicKey = "new-key"
+			fakeInformerBackupEntry.Update(bastion1Copy, bastion1)
+			lock.Lock()
+			defer lock.Unlock()
+			paths[VertexTypeBackupEntry] = append(paths[VertexTypeBastion], pathExpectation{VertexTypeBastion, bastion1.Namespace, bastion1.Name, VertexTypeSeed, "", seed1.Name, BeTrue()})
```

Comment:
```suggestion
			paths[VertexTypeBastion] = append(paths[VertexTypeBastion], pathExpectation{VertexTypeBastion, bastion1.Namespace, bastion1.Name, VertexTypeSeed, "", seed1.Name, BeTrue()})
```

(I guess this is the reason for the flakes you observe)